documentation/source/release_notes.rst

```diff
@@ -4,6 +4,36 @@ Release Notes
 
 All releases are available from the `GitHub Releases Page <https://github.com/cocotb/cocotb/releases>`_.
 
+cocotb 1.3
+==========
+
+Released on 1 January 2020
+
+New features
+------------
+- :func:`cocotb.triggers.with_timeout`, for a shorthand for waiting for a trigger with a timeout. (:pr:`1119`)
+- New ``timeout_time`` and ``timeout_unit`` arguments to :func:`cocotb.test`, for adding test timeouts. (:pr:`1119`)
+- The ``expect_error`` argument to :func:`cocotb.test` now accepts a specific exception type. (:pr:`1116`)
+- :envvar:`COCOTB_RESULTS_FILE`, to allow configration of the xunit XML output filename.  (:pr:`1053`)
+- :class:`cocotb.triggers.Timer` can now be used with :class:`fraction.Fraction` instances, allowing constructs like ``Timer(Fraction(1, 10000), units='sec')`` as an alternate spelling for ``Timer(100, units='us')``. (:pr:`1114`)
```

Comment:
Should this be
```suggestion
- :class:`cocotb.triggers.Timer` can now be used with :class:`fractions.Fraction` instances, allowing constructs like ``Timer(Fraction(1, 10000), units='sec')`` as an alternate spelling for ``Timer(100, units='us')``. (:pr:`1114`)
```
? (https://docs.python.org/3/library/fractions.html#fractions.Fraction)